test/evergreen.yml

```diff
@@ -1208,7 +1208,7 @@ variables:
           # See https://bugzilla.redhat.com/show_bug.cgi?id=1686261#c10 for the potential cause.
           format_test_script_args: -t 360 -- -C "mmap=false,mmap_all=false"
           test_env_vars:
-            ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0:unmap_shadow_on_exit=1"
+            ASAN_OPTIONS="abort_on_error=1:detect_leaks=1:disable_coredump=0:unmap_shadow_on_exit=1"
```

Comment:
As a separate ticket, we could look at using the variables section to define this once and then reuse it across the variants (see something like configure_flags_with_builtins). It might go with things like the PATH as well.

Reply:
Not required for this PR, but could be a good improvement.

Reply:
@mickgraham-mongodb Yes the symbolizer path and common SAN flags should be in some kind of yaml fragment that can be referred to, but I'm unfamiliar with Evergreen yaml that implementing this would take too much time atm, created: [WT-11298](https://jira.mongodb.org/browse/WT-11298).

Reply:
Thanks